microcoap/Makefile

```diff
@@ -0,0 +1,34 @@
+# name of your application
+APPLICATION = microcoap-example
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../../RIOT
+
+# Comment this out to disable code in RIOT that does safety checking
+# which is not needed in a production environment but helps in the
+# development process:
+CFLAGS += -DRIOT -DMICROCOAP_DEBUG #-DENABLE_DEBUG
+
+# Change this to 0 show compiler invocation lines by default:
+QUIET ?= 1
+
+USEPKG=microcoap
+
+USEMODULE += config
+USEMODULE += shell
+USEMODULE += shell_commands
+USEMODULE += uart0
+
+ifeq ($(strip $(BOARD)),native)
+	USEMODULE += nativenet
+else
+	USEMODULE += cc110x
+endif
```

Comment:
Actually, when I think about it, does it even make sense to include anything but native for the example? Having just native makes for a much slimmer Makefile, and I can't guarantee (yet) that it will work on any proper boards anyway. If you think this application should reflect our experiences (i.e. we definitely got this to work on the SAMR21, or something like that), I'd add the lines for that as soon as I'm confident about them.  
TL;DR: I propose to get rid of anything but native for now.